ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ApprovedDeferrals.tsx

```diff
@@ -116,88 +117,93 @@ function ApprovedDeferrals() {
     }
 
     return (
-        <PageSection>
-            <Toolbar>
-                <ToolbarContent>
-                    <FilterAutocompleteSelect
-                        searchFilter={searchFilter}
-                        onFilterChange={(newFilter) => setSearchFilter(newFilter)}
-                        searchOptions={searchOptions}
-                    />
-                    <ToolbarItem variant="pagination" alignment={{ default: 'alignRight' }}>
-                        <Pagination
-                            toggleTemplate={({ firstIndex, lastIndex }) => (
-                                <span>
-                                    <b>
-                                        {firstIndex} - {lastIndex}
-                                    </b>{' '}
-                                    of <b>many</b>
-                                </span>
-                            )}
-                            page={page}
-                            perPage={perPage}
-                            onSetPage={(_, newPage) => setPage(newPage)}
-                            onPerPageSelect={(_, newPerPage) => setPerPage(newPerPage)}
-                            isCompact
+        <>
```

Comment:
Observing, not necessarily suggesting, that `PageTitle` uses react-helmet for side effect to update `title` in `head` not any change in `body` element, therefore you can render it as a child instead of a sibling of `PageTitle` element to save an indentation change.

Reply:
Good point. I guess it doesn't matter if I did it this way or just had it under `<PageSection>`. It's still somewhere within the hierarchy of components in the `body`. I think it would look cleaner if I didn't use the `<></>` so I'll make a change